contrib/gplate/geojson2json.ipynb

```diff
@@ -0,0 +1,108 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Geojson2json.py is a simple script that directly converts GeoJSON files extracted using the GPlates GUI to the World Builder-compatible JSON format. Currently, it supports continental plate geometries. \n",
```

Comment:
The problem here is not that you need to change geojson to json (geojson is a compatible flavor of json), it is that you want to translate directly to a json flavor (.wb) that is readable by the World Builder. This change would make that more clear:

```suggestion
    "Geojson2json.py is a simple script that directly converts GeoJSON files extracted using the GPlates GUI to the World Builder-compatible .wb format. Currently, it supports continental plate geometries. \n",
```